Pod/ILGClasses/ILGClasses.h

```diff
@@ -8,6 +8,8 @@
 
 #import <Foundation/Foundation.h>
 
+NS_ASSUME_NONNULL_BEGIN
```

Comment:
If it's ever used anywhere in a project and not necessarily used everywhere, it makes it impossible to understand nullability of parameters from a declaration alone without having to check up and down to see if nullability defaults are inverted.  It's the programming equivalent of "Nope, it's opposite day."